pages/bal/voie.js

```diff
@@ -15,12 +15,16 @@ import useFuse from '../../hooks/fuse'
 import TableRow from '../../components/table-row'
 import VoieEditor from '../../components/bal/voie-editor'
 import NumeroEditor from '../../components/bal/numero-editor'
+import DeleteWarning from '../../components/delete-warning'
+import GroupedActions from '../../components/grouped-actions'
 
 const Voie = React.memo(({voie, defaultNumeros}) => {
   const [editedVoie, setEditedVoie] = useState(null)
   const [isEdited, setEdited] = useState(false)
   const [hovered, setHovered] = useState(false)
   const [isAdding, setIsAdding] = useState(false)
+  const [error, setError] = useState()
+  const [toRemove, setToRemove] = useState(null)
```

Comment:
Il est préférable d'utiliser de préfixer le nom des variables booléenne par `is`. Exemple `[isRemoveWarningShown, setIsRemoveWarningShown]`